### PR TITLE
Fix for bitly.com

### PR DESCRIPTION
**Pull request by Rakleed**
- Remove background.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -982,6 +982,15 @@ span.code {
 
 ================================
 
+bitly.com
+
+CSS
+.hero-content {
+    background: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:989]: Use `background-image`

Comment (Rakleed) [reply src/config/dynamic-theme-fixes.config:989]: Why?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:989]: It should specify the CSS property.

Comment (Rakleed) [reply src/config/dynamic-theme-fixes.config]: Ok, fixed.

**Pull request merged**